src/demo/share/java2d/J2DBench/src/j2dbench/tests/iio/OutputImageTests.java

```diff
@@ -143,7 +143,7 @@ private static void initIIOWriteFormats() {
             String klass = spi.getClass().getName();
             String format = spi.getFormatNames()[0].toLowerCase();
             String suffix = spi.getFileSuffixes()[0].toLowerCase();
-            if (suffix == null || suffix.equals("")) {
```

Comment:
This file intentionally maintains compatibility to jdk1.4, so equals("") should be used.